src/transformStateWithClones.js

```diff
@@ -7,7 +7,38 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const stateVersions = [];
+  let controllState = { ...state };
```

Comment:
it would be better to specify that our variable stores a **copy** of our state, the word `control` does not give us a clear understanding of what is stored in this variable